docs/ui/insight/tools/search.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Search capabilities"
+linkTitle: Search
+description: Learn how RedisInsight allows you to index and perform search queries
+weight: 10
+aliases:
+---
+
+If you're using Redis Stack's indexing, querying, or full-text search capabilities, RedisInsight provides UI controls to run search queries against a preselected index. You can also create a secondary index of your data in a dedicated pane.
+
```

Comment:
Search for keywords and phrases within your documents stored as Redis [hashes](/docs/manual/data-types/#hashes) or [JSON](/docs/stack/json/) with dedicated UI controls to run search queries against a pre-selected index.
If you use the [Search](docs/stack/search/) capability and would like to use dedicated UI in RedisInsight, then open the search by values of keys in Browser, select your index, insert the search query, and run it. After that, RedisInsight will display the list of keys found in List or Tree view.

RedisInsight also provides UI controls to create a new secondary index of your data where you need to specify basic index information in order to create one.